.github/workflows/python-package.yml

```diff
@@ -7,8 +7,14 @@ on:
   workflow_dispatch:
   push:
     branches: [ main ]
+    paths-ignore:
```

Comment:
about: paths-ignore https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths